@kiva/kv-shop/src/oneTimeCheckout.ts

```diff
@@ -8,7 +8,8 @@ import { ShopError, parseShopError } from './shopError';
 import { callShopMutation, callShopQuery } from './shopQueries';
 import { validatePreCheckout } from './validatePreCheckout';
 import { wait } from './util/poll';
-import getVisitorID from './util/visitorId';
+import { getVisitorID } from './util/visitorId';
```

Comment:
I can't comment directly on the line, but GH is saying `Unable to resolve path to module '@kiva/kv-analytics'`

Reply:
Yeah, we've been having that error since [this PR](https://github.com/kiva/kv-ui-elements/pull/323#issuecomment-1836750401). We think maybe gha has a cached version of @kiva/kv-analytics? If you have any ideas what might be wrong there or how to fix it up, please share!

Reply:
Note there a separate import from kv-analytics that shows the same lint error for me locally... different method I think